assets/docs/callmemehdi/models/AraBERT/1.md

```diff
@@ -1,6 +1,6 @@
 # Module callmemehdi/arabert/1
 
-AraBERT is an Arabic pretrained lanaguage model based on Google's BERT architechture.
+AraBERT is an Arabic pretrained lanaguage model based on Google's BERT base architechture.
```

Comment:
nit: architecture

Reply:
I'm sorry Morgan I didn't get this remark.

Reply:
Just a small typo in architecture. You have an extra h part way through the word.